common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/js-bridge.js

```diff
@@ -0,0 +1,205 @@
+        if (pendingResolveCreate === null || pendingRejectCreate === null) {
+            console.log("Reply failure: Resolve: " + pendingResolveCreate +
+            " and reject: " + pendingRejectCreate);
+            return;
+        }
+
+        if (reply.status != 'success') {
+            var reject = pendingRejectCreate;
+            pendingResolveCreate = null;
+            pendingRejectCreate = null;
+            reject(new DOMException(reply.data.domExceptionMessage, reply.data.domExceptionName));
+            return;
+        }
+        var cred = credentialManagerDecode(reply.data);
+        var resolve = pendingResolveCreate;
+        pendingResolveCreate = null;
+        pendingRejectCreate = null;
+        resolve(cred);
+    }
+    __webauthn_hooks__.onReplyCreate = onReplyCreate;
+    /**
+     * This decodes the output from the credential manager flow to parse back into URL format. Both
+     * get and create flows ultimately return a PublicKeyCredential object.
+     * @param json_result
+     */
+    function credentialManagerDecode(decoded_reply) {
+        decoded_reply.rawId = CM_base64url_decode(decoded_reply.rawId);
+        decoded_reply.response.clientDataJSON = CM_base64url_decode(decoded_reply.response.clientDataJSON);
+        if (decoded_reply.response.hasOwnProperty('attestationObject')) {
+            decoded_reply.response.attestationObject = CM_base64url_decode(decoded_reply.response.attestationObject);
+        }
+        if (decoded_reply.response.hasOwnProperty('authenticatorData')) {
+            decoded_reply.response.authenticatorData = CM_base64url_decode(decoded_reply.response.authenticatorData);
+        }
+        if (decoded_reply.response.hasOwnProperty('signature')) {
+            decoded_reply.response.signature = CM_base64url_decode(decoded_reply.response.signature);
+        }
+        if (decoded_reply.response.hasOwnProperty('userHandle')) {
+            decoded_reply.response.userHandle = CM_base64url_decode(decoded_reply.response.userHandle);
+        }
+        decoded_reply.getClientExtensionResults = function getClientExtensionResults() { return {}; };
+        decoded_reply.response.getTransports = function getTransports() {
+            if (decoded_reply.response.hasOwnProperty('transports')) { return decoded_reply.response.transports; }
+            return [];
+        };
+        return decoded_reply;
+    }
+})(__webauthn_hooks__ || (__webauthn_hooks__ = {}));
+__webauthn_hooks__.originalGetFunction = navigator.credentials.get;
+__webauthn_hooks__.originalCreateFunction = navigator.credentials.create;
+navigator.credentials.get = __webauthn_hooks__.get;
+navigator.credentials.create = __webauthn_hooks__.create;
+// Some sites test that `typeof window.PublicKeyCredential` is
+// `function`.
+window.PublicKeyCredential = (function () { });
+window.PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable =
+    function () {
+        return Promise.resolve(false);
```

Comment:
Just double checking my understanding: this part controls if the passkey option is visible correct? By always returning false here, this ensures that we can control the visibility with webauthn=1 instead?